src/scripts.js

```diff
@@ -116,13 +111,13 @@ function searchRecipe() {
   } else if (nameSearched.length > 0) {
     viewAllRecipes(nameSearched);
   } else {
-    viewAllRecipes(recipeRepo.repo)
+    viewAllRecipes(recipeRepo.repo);
   }
 }
 
 function searchFavoriteRecipe() {
   if (!searchBox.value) {
-    viewAllRecipes(currentUser.favoriteRecipes)
+    viewAllRecipes(currentUser.favoriteRecipes);
   }
   const tagSearched = currentUser.filterFavoriteRecipeTag(searchBox.value);
   const nameSearched = currentUser.filterFavoriteRecipeName(searchBox.value);
```

Comment:
Down here it's the opposite. If you add a few items and search by their tag they might come up even if the search term isn't in the title. I'd either make these consistent or update this functionality to push into an array if the search term is in the title of the recipe _or_ the tags of the recipe. And then maybe add "Search by title or tag" in the search bar placeholder text so it's clear to the user.